.github/workflows/assets/swagger.yaml

```diff
@@ -0,0 +1,2697 @@
+openapi: 3.0.1
```

Comment:
I switched to `.yaml` and also added in anything missing that was added to ext-documentation repo. Still left out `allOf` to not break anything.